src/utils/index.ts

```diff
@@ -1117,7 +1117,7 @@ export function createClaim(
     return {
       type,
       // this assertion is necessary because CountryCode is not in the middleware types
-      code: jurisdiction as CountryCode,
+      code: stringUpperFirst(jurisdiction?.toLowerCase()) as CountryCode,
```

Comment:
```suggestion
      code: stringUpperFirst(jurisdiction!.toLowerCase()) as CountryCode,
```

To reflect that we don't ever expect `jurisdiction` to be falsy in this scenario